owa/app/js/components/manage_patient_identifier_sources/new_patient_identifier_sources/localIdentifierGenerator.jsx

```diff
@@ -0,0 +1,85 @@
+import React, {Component} from 'react';
+import {Table, Button} from 'reactstrap';
+import './newPatientIdentifierSources.css'
+
+export default class LocalIdentifierGenerator extends Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+
+    return (
+      <div>
+        <div>New: Local Identifier Generator for {this.props.title}</div>
+        <table>
+          <thead>
+            <tr>
+              <td>Field</td>
+              <td>Value</td>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td><span className="requiredField">*</span> Name:</td>
```

Comment:
I like this concept of using the same class for all required fields - `requiredField`. Never thought of that 😄